### PR TITLE
Use correct amount in `sendTransaction`

### PR DESCRIPTION
**Pull request by kchulj**
The actual `Amount` that is sent is 10,000,000 times the original amount.
This amount includes the number of "cents".

This is similar to [`AmountFmt` in `PrettyPrinter`](https://github.com/bosagora/agora/blob/68df58ee57851017ccbd73da1b4c98000088a672/source/agora/utils/PrettyPrinter.d#L141-L170)

### CODE REVIEW

File: frontend/src/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h1 class="h3 mb-3 fw-normal">Welcome to the BOSAGORA Faucet</h1>
             </div>
             <br>
             <div class="form-floating">
-              <input name="amount" type="text" class="form-control" id="floatingInput" value="100" readonly>
+              <input name="amount" type="text" class="form-control" id="floatingInput" value="1000000000" readonly>
```
Comment (Geod24) [inline frontend/src/index.html:28]: Or just use the number of coins here?

File: source/faucet/main.d [hunk 2]
```diff
@@ -340,7 +340,7 @@ public class Faucet : FaucetAPI
         if (leftover <= first_utxo.value.output.value)
         {
             Transaction tx = txb.draw(leftover, [pubkey]).sign();
-            logInfo("Sending %s BOA to %s", amount, recv);
+            logInfo("Sending %s BOA to %s", amount / 10_000_000, recv);
```
Comment (Geod24) [inline source/faucet/main.d:343]: https://github.com/bosagora/agora/blob/68df58ee57851017ccbd73da1b4c98000088a672/source/agora/common/Amount.d#L372-L394

**Review by Geod24 — CHANGES_REQUESTED** [review]

File: frontend/src/index.html (hunk 1 above)
Comment (kchulj) [reply frontend/src/index.html:28]: So keep it at 100?

**Comment by kchulj** [comment]
Ready for review.

**Review by linked0 — APPROVED** [review]
LGTM

**Review by Geod24 — APPROVED** [review]

**Pull request merged**